### DISCUSSION TITLE
(FEE) HELP PLEASE! I MAKE A BIG MISTAKE!!!!!

### DISCUSSION THREAD

**Discussion by jmloujedo** [Chia-Network/chia-blockchain#4700] [Support]
Hello, I made a serious mistake, I need you to help me please, in my transaction history you can corroborate that what I say is true, I wanted to send 2 (XCH) to my de-centralized wallet I put the amount of 2 and in (FEE) I put 1 I didn't want to put 1, would you please return it to me or even a percentage? it is obvious that nobody would put that amount having 2 XCH. I would really appreciate your help, hugs.

JuanMa

My Wallet: xch1allrv3yu28mg560ul3a93490v0pktjug078gsvq3cfhtc0sxy80skv355j

![History](https://user-images.githubusercontent.com/73976103/117998530-5dd9ab80-b31a-11eb-88c1-a64d28eb78b9.PNG)

**Comment by keeperlink** [comment]
Wow, that's unfortunate...
We really need to have a big warning prompt in UI when someone accidentally put unreasonably high transaction fee

**Comment by FSys77** [comment]
The farmer who farmed the block must be very happy now :-D

**Comment by jmloujedo** [comment]
Who is the farmer ? i want to recover that XCH is so much money for mi and
here in argentina is not easy to recover that money.

**Reply by reflex440** [reply]
This is sadly an expensive lesson. I empathize brother, I have made errors in the past too.

**Comment by oytp** [comment]
I think this is impossible. Be more accurate next time

**Comment by jmloujedo** [comment]
or maybe you put some advice or warning thanks for nothing.

**Comment by jmloujedo** [comment]
Sorry about that , I was ungry, the truth is that you guys are the best. I
love you all, this is the best project ever.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
